cwms-data-api/src/main/java/cwms/cda/ApiServlet.java

```diff
@@ -703,6 +705,7 @@ protected void configureRoutes() {
         addProjectLocksHandlers("/project-locks/{name}", requiredRoles);
         addProjectLockRightsHandlers("/project-lock-rights/{project-id}", requiredRoles);
 
+        get("/location-kinds/", new LocationKindController(metrics));
```

Comment:
```suggestion
        get("/location/by-kinds/", new LocationKindController(metrics));
```

Okay, one last nitpick. This as the name?

It's not a location-kind list, it's a list of locations with only the office, name, and kind that allows some searching.

@rma-rripken

Reply:
That more accurately represents the usage/intent. Perhaps `/location-with-kind/`?

Reply:
`locations/with-kind`

This shouldn't be a root endpoint. it's behavior under locations.

Reply:
Changed endpoint path to `/locations/with-kind/`

Reply:
```/locations/with-kind/``` seems like a good place